### ISSUE TITLE
flatlaf.useRoundedPopupBorder for kde?

### ISSUE THREAD

**Issue by Shadowblitz16** [JFormDesigner/FlatLaf#949]
Not sure if it's possible but it would be nice to see flatlaf.useRoundedPopupBorder supported on kde 5 and up

I am using kubuntu 24.04 which is linux.

**Comment by DevCharly** [comment]
That would be indeed nice, but, how to do it for Java apps, which does not use KDE.

Not even JetBrains has managed to implement this yet for IntelliJ IDEA: https://youtrack.jetbrains.com/issue/JBR-7476

**Comment by DevCharly** [comment]
Please try PR #964

**Comment by Shadowblitz16** [comment]
> Please try PR [#964](https://github.com/JFormDesigner/FlatLaf/pull/964)

Call me a bad developer but idk how to use PR's

**Comment by DevCharly** [comment]
> > Please try PR [#964](https://github.com/JFormDesigner/FlatLaf/pull/964)
> 
> Call me a bad developer but idk how to use PR's

see https://github.com/JFormDesigner/FlatLaf/pull/964#issuecomment-2616402045

**Comment by MrUrdam** [comment]
Is this PR covering GNOME as well? Or is it for now just a KDE based solution? :)

**Comment by DevCharly** [comment]
> Is this PR covering GNOME as well? Or is it for now just a KDE based solution? :)

Yes. It is a pure Java solution and does not use or depend on GNOME or KDE.

**Comment by DevCharly** [comment]
@MrUrdam thanks for testing.
Are there any differences in font rendering, in the rounded popup, compared to FlatLaf 3.5.4?

**Comment by MrUrdam** [comment]
Hey @DevCharly I just did a short test last week and rolled back to the current version we are using. We are still not on the latest FlatLaf version. 
In my short investigation back then I have not seen any issues or weird glitches. 

Would it be possible to add a option to round the corners of the frame via FlatLaf as well? Here it would make sense that the Dev can decide which corners should be rounded as well. In GNOME for example only the top corners are rounded with the when using the System Decirations. With the FlatLaf Window Decorations we could then change the UI to a similar L&F

**Comment by DevCharly** [comment]
> Would it be possible to add a option to round the corners of the frame via FlatLaf as well?

Yes, that's the next step 😉 

> Here it would make sense that the Dev can decide which corners should be rounded as well.

Don't think that this is necessary (or worth the effort).
Latest GNOME versions (e.g. on Ubuntu 24.10) are rounded on **all four** corners.
